odml/tools/dict_parser.py

```diff
@@ -280,7 +317,14 @@ def parse_sections(self, section_list):
                     # Make sure to always use the correct odml format attribute name
                     sec_attrs[odmlfmt.Section.map(attr)] = content
 
-            sec = odmlfmt.Section.create(**sec_attrs)
+            try:
+                sec = odmlfmt.Section.create(**sec_attrs)
+            except Exception as exc:
+                msg = "Section not created (%s)\n  %s" % (sec_attrs, str(exc))
+                self.error(msg)
+                # If recovered in ignore_error mode, return empty list
```

Comment:
Not an issue but I guess the list will only be empty if it's the first section right?

Could this do the same as the `parse_properties()` method and continue with the rest of the sections instead of returning?

Reply:
Yes, good catch; that's what it should do.